content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -193,7 +194,7 @@ sudo apt-get install -y kubelet kubeadm kubectl
 sudo apt-mark hold kubelet kubeadm kubectl
 ```
 {{% /tab %}}
-{{% tab name="CentOS, RHEL or Fedora" %}}
+{{% tab name="CentOS, RHEL, Fedora, or Amazon Linux" %}}
```

Comment:
suggesting these tab names:

```
{{% tab name="Debian based distributions" %}}
{{% tab name="RedHat based distributions" %}}
{{% tab name="Without a package manager" %}}
```

alternatively we can talk about the package managers instead of "Debian" and "RedHat"

Reply:
Very good. Personally, I think saying Debian and "Red Hat" (I'll make it two words) gives the right message. And saying "Without a package manager" for the third makes the right distinction. There are still RPM packages in Fedora CoreOS, I think. It's just not set up to use yum.

Reply:
Done.